plover/gui_qt/main_window.py

```diff
@@ -238,6 +243,15 @@ def on_disable_output(self):
     def on_configure(self):
         self._configure()
 
+    def on_open_config_folder(self):
+        # webbrowser.open does not work on Mac so we need OS-specific logic
+        if sys.platform.startswith('win'):
+            os.startfile(CONFIG_DIR)
+        elif sys.platform.startswith('linux'):
+            subprocess.run(['xdg-open', CONFIG_DIR])
```

Comment:
For Python 3.4 compatibility:
```suggestion
            subprocess.call(['xdg-open', CONFIG_DIR])
```